.eleventy.js

```diff
@@ -128,6 +128,9 @@ module.exports = function(eleventyConfig) {
     eleventyConfig.addPassthroughCopy({ "./node_modules/asciinema-player/resources/public/js/asciinema-player.js": "./asciinema-player.js" });
     eleventyConfig.addPassthroughCopy({ "./node_modules/asciinema-player/resources/public/css/asciinema-player.css": "./asciinema-player.css" });
 
+    // Pass through the redirects
+    eleventyConfig.addPassthroughCopy({ "./redirects.tsv": "./_redirects" });
```

Comment:
Added config line to tell 11ty to copy `redirects.tsv` to `_site/_redirects`

11ty Passthrough File Copy: https://www.11ty.dev/docs/copy/